packages/react/.storybook/preview.js

```diff
@@ -15,25 +12,25 @@ configureActions({
   depth: 3,
 });
 
-addParameters({
+export const parameters = {
+  docs: {
+    container: DocsContainer,
+    page: DocsPage,
+  },
   options: {
-    theme: theme,
-    showRoots: true,
     storySort: (a, b) =>
       a[1].kind.replace(/☢️-|🚫-|⚠️-/i, '') === b[1].kind.replace(/☢️-|🚫-|⚠️-/i, '')
         ? 0
         : a[1].id
             .replace(/☢️-|🚫-|⚠️-/i, '')
             .localeCompare(b[1].id.replace(/☢️-|🚫-|⚠️-/i, ''), undefined, { numeric: true }),
   },
-});
-
-addParameters({
-  docs: {
-    container: DocsContainer,
-    page: DocsPage,
+  a11y: {
+    element: '#root',
+    config: {},
+    options: {},
+    manual: true,
   },
-});
-addDecorator((story) => <Container story={story} />);
-addDecorator(withA11y);
-addDecorator(withKnobs);
+};
+
+export const decorators = [(story) => <Container story={story} />, withKnobs];
```

Comment:
addDecorator is deprecated and a single exported array of function is used instead